### PR TITLE
Cli: Trace

### PR DESCRIPTION
**Pull request by alanmarkoTrilitech**
# Context

https://app.asana.com/0/1205170694555918/1205507584925473

# Description
Adds trace command to the CLI - works similarly to ./view-console.sh.

# Manually testing the PR
`cargo run -- logs trace`

### CODE REVIEW

**Review by johnyob — CHANGES_REQUESTED** [review]
Implementation of `trace` LGTM. 

Could you please rename this command to `jstz logs trace` to be more inline with our existing design document on logging.

File: jstz_cli/src/main.rs [hunk 1]
```diff
@@ -5,10 +5,12 @@ mod account;
 mod bridge;
 mod config;
 mod deploy;
+mod logs_command;
```
Comment (johnyob) [inline jstz_cli/src/main.rs:8]: `logs_command` -> `logs`, fits better w/ existing naming

Comment (alanmarkoTrilitech) [reply jstz_cli/src/main.rs:8]: The main reason why I didn't like it was that logs is a special folder in VS Code making it a bit confusing.

Comment (alanmarkoTrilitech) [reply jstz_cli/src/main.rs:8]: And it is by default ignored in .gitignore

**Review by johnyob — APPROVED** [review]

**Pull request merged**